common/djangoapps/student/models.py

```diff
@@ -1028,7 +1028,7 @@ class CourseEnrollmentManager(models.Manager):
     Custom manager for CourseEnrollment with Table-level filter methods.
     """
 
-    def num_enrolled_in(self, course_id):
+    def num_enrolled_in(self, course_id, max_enrollment_for_buttons):
```

Comment:
@Ali-D-Akbar are we sure this method is not used anywhere else? Please double-check this.

Reply:
Yup, there's only ONE.

<img width="1002" alt="Screenshot 2020-07-01 at 5 36 06 PM" src="https://user-images.githubusercontent.com/52413434/86244395-58bbb400-bbc1-11ea-88dd-b661b29c5fd4.png">